### PR TITLE
Client changes for other collection pages

### PR DESCRIPTION
**Pull request by stef359**
Sorry I focused only on the NCYTE page that I forgot to make necessary changes for the other collection pages,

**NICE:**
Screenshot 2024-07-31 at 9 31 57 PM

**502:**
Screenshot 2024-07-31 at 9 54 57 PM

**General collection pages:**
Screenshot 2024-07-31 at 9 47 10 PM

NCYTE:
Screenshot 2024-07-31 at 10 18 32 PM

### CODE REVIEW

File: src/app/core/feature-module/featured.routes.ts [hunk 1]
```diff
@@ -17,6 +17,15 @@ export const FEATURED_ROUTES = {
   GET_COLLECTION_FEATURED_OBJECTS(collectionAbvName: string) {
     return `${environment.apiURL}/featured/learning-objects?collection=${encodeURIComponent(collectionAbvName)}`;
   },
+  /**
+  * Request to retrieve the featured learning objects for a specific collection with a limit
+  * @method GET
+  * @param collectionAbvName the abbreviated name of the collection
```
Comment (shanioffe) [inline src/app/core/feature-module/featured.routes.ts:23]: Nit: add the limit param

**Review by mfranz1 — COMMENTED** [review]
Great Job! Thank you! I do have a requested change on your API that will fix the NCyTE collection card

**Review by mfranz1 — APPROVED** [review]

**Review by shanioffe — APPROVED** [review]

**Comment by pzalep1** [comment]
The NICE page is returning all featured learning objects not just the NICE Challenge ones.

**Review by pzalep1 — CHANGES_REQUESTED** [review]
Just the note about the NICE Challenge page.

**Comment by stef359** [comment]
@pzalep1 All the featured learning objects in the nice challenge page are confirmed nice-challenge learning objects. I just added a lot of them to the featured collection so I can test out the limit parameter being used by the route. Just had to make sure that if there were 7 nice objects in the featured collection only 6 would still show up.

**Comment by pzalep1** [comment]
> @pzalep1 All the featured learning objects in the nice challenge page are confirmed nice-challenge learning objects. I just added a lot of them to the featured collection so I can test out the limit parameter being used by the route. Just had to make sure that if there were 7 nice objects in the featured collection only 6 would still show up.

The Zero Trust learning objects in your screenshot are in the NCAE-C collection. Can you update that screenshot with the NICE objects to confirm?

**Review by pzalep1 — CHANGES_REQUESTED** [review]
Response to the screenshot.

**Review by pzalep1 — APPROVED** [review]

**Review by curlyLasagna — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**